cargo-workspaces/src/utils/version.rs

```diff
@@ -219,6 +248,62 @@ impl VersionOpt {
         Ok(())
     }
 
+    fn alert_unversioned(
+        &self,
+        pkgs: HashMap<&str, (&str, Vec<(&str, &VersionReq, &Version)>)>,
+    ) -> Result {
+        if pkgs.is_empty() {
+            return Ok(());
+        }
+        loop {
+            match Select::with_theme(&ColorfulTheme::default())
+                .with_prompt(format!(
+                    "You have {} packages with unversioned dependencies",
+                    pkgs.len()
+                ))
+                .items(&["Review Dependencies", "Auto-version", "Abort"])
```

Comment:
Do we want to let them proceed forward without auto versioning?

Reply:
Will do.

Reply:
Done.

So now, “Skip” just defers the former, familiar behavior, but I'm thinking, should we add another confirmation informing users that without explicit interdependency versioning, (which could be aided by auto-versioning), `cargo package` and hence, `cargo publish` won't work.